airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -441,6 +441,25 @@ def filter_null_bytes(self, b: bytes):
             self.logger.warning("Filter 'null' bytes from string, size reduced %d -> %d chars", len(b), len(res))
         return res
 
+    def get_response_encoding(self, headers) -> str:
```

Comment:
can you add unit tests for this method? We should test
1. headers without a `content-type` field -> `returns self.encoding`
2. headers that does not produce a charset in `params` (you know better than me what scenario would lead to this) -> returns `self.encoding`
3. headers with a charset in `params` -> returns `params["charset"].strip("'\"")`

Reply:
https://github.com/airbytehq/airbyte/pull/29622